### PR TITLE
metrics: fix inconsistent label for panic counter

### PR DESCRIPTION
**Pull request by coocood**
The declared number of labels should be the used number of labels.

### CODE REVIEW

File: metrics/metrics_test.go [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+	TestingT(t)
+}
+
+var _ = Suite(&testSuite{})
+
+type testSuite struct {
+}
+
+func (s *testSuite) TestMetrics(c *C) {
+	PanicCounter.WithLabelValues(LabelDomain).Inc()
```
Comment (shenli) [inline metrics/metrics_test.go]: What is the purpose of this test case?

Comment (coocood) [reply metrics/metrics_test.go]: To test it doesn't panic when called.

Comment (shenli) [reply metrics/metrics_test.go]: Please comment it. I am confused by this test.

**Comment by coocood** [comment]
@shenli PTAL

**Comment by shenli** [comment]
LGTM

**Comment by shenli** [comment]
@zhexuany PTAL

**Pull request merged**